dapps/shop/src/pages/super-admin/Settings.js

```diff
@@ -53,7 +54,7 @@ const SuperAdminSettings = () => {
     }).then(async res => {
       if (res.ok) {
         const json = await res.json()
-        setState(json)
+        setState(json.config)
```

Comment:
Hrm not sure this is necessary any more?  I made a recent change that lumps it all into one object

Reply:
Missed that.  Haven't tested with the recent master changes.  Let me do that before merge.

Reply:
Reverted this.  Looks to be working.